src/TLuaInterpreter.cpp

```diff
@@ -10217,86 +10217,151 @@ int TLuaInterpreter::getCmdLine(lua_State* L)
 
 int TLuaInterpreter::installPackage(lua_State* L)
 {
-    string event;
+    QString packageName;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "installPackage(): wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "installPackage: bad argument #1 type (pathFileName of package as string expected, got %s!)",
```

Comment:
Better as `package path as string expected`

Reply:
See next comment below.

Reply:
I've got a compromise of `path and file name as string` - which does explain why the requirement is different between _loading_ a package/module where the physical file has to be located in the file-system and _unloading_ where the package/module __name__ is required (and in the case of modules the module name _need not be the same_ as the file name).